app-header/app-header.html

```diff
@@ -95,6 +95,8 @@
       position: relative;
       display: block;
       -webkit-transform: translateZ(0);
```

Comment:
that was added to fix an issue in Safari related to the waterfall shadow, but it's unfortunate that I didn't leave a comment.